app/components/primer/alpha/layout.pcss

```diff
@@ -0,0 +1,268 @@
+/* Layout */
+
+@define-mixin flow-as-row {
+  grid-auto-flow: row;
+  grid-template-columns: 1fr !important;
+
+  & .Layout-sidebar,
+  & .Layout-divider,
+  & .Layout-main {
+    width: 100% !important;
+    grid-column: 1 !important;
+  }
+
+  &.Layout--sidebarPosition-flowRow-start {
+    & .Layout-sidebar {
+      grid-row: 1;
+    }
+
+    & .Layout-main {
+      grid-row: 2 / span 2;
+    }
+  }
+
+  &.Layout--sidebarPosition-flowRow-end {
+    & .Layout-sidebar {
+      grid-row: 2 / span 2;
+    }
+
+    & .Layout-main {
+      grid-row: 1;
+    }
+  }
+
+  &.Layout--sidebarPosition-flowRow-none {
+    & .Layout-sidebar {
+      display: none;
+    }
+  }
+
+  &.Layout--divided {
+    @mixin flow-as-row-divider;
+
+    & .Layout-main {
+      grid-row: 3 / span 1;
+    }
+
+    &.Layout--sidebarPosition-flowRow-end {
+      & .Layout-sidebar {
+        grid-row: 3 / span 1;
+      }
+
+      & .Layout-main {
+        grid-row: 1;
+      }
+    }
+  }
+}
+
+@define-mixin flow-as-row-divider {
+  --Layout-gutter: 0;
+
+  & .Layout-divider {
+    height: 1px;
+    grid-row: 2;
+
+    &.Layout-divider--flowRow-hidden {
+      display: none;
+    }
+
+    &.Layout-divider--flowRow-shallow {
+      height: 8px;
+      margin-right: 0;
+      background: var(--color-canvas-inset);
+      border-color: var(--color-border-default);
+      border-style: solid;
+      border-width: $border-width 0;
+    }
+  }
+}
+
+.Layout {
+  display: grid;
+
+  --Layout-sidebar-width: 220px;
+  --Layout-gutter: 16px;
+
+  @media (max-width: calc(544px - 0.02px)) {
+    @mixin flow-as-row;
+  }
+
+  &.Layout--flowRow-until-md {
+    @media (--primer-viewportRange-narrow-viewport) {
```

Comment:
It seems this is the only token that has a 1:1 replacement [from Primitives](https://unpkg.com/browse/@primer/primitives@7.8.3/tokens-v2-private/css/tokens/functional/size/viewport.css). 

```css
@custom-media --primer-viewportRange-narrow-viewport (max-width: calc(48rem - 0.02px));
@custom-media --primer-viewportRange-narrowLandscape-viewport (max-width: calc(63.25rem - 0.02px) and (max-height: calc(34rem - 0.02px)) and (orientation: landscape));
@custom-media --primer-viewportRange-regular-viewport (min-width: 48rem);
@custom-media --primer-viewportRange-wide-viewport (min-width: 90rem);
@custom-media --primer-viewportRange-portrait-viewport (orientation: portrait);
@custom-media --primer-viewportRange-landscape-viewport (orientation: landscape);
```

So I hard coded the rest. Here the migration guide so far:

In px | In rem | Token
--- | --- | ---
`max-width: calc(544px - 0.02px)` | `max-width: calc(34rem - 0.02px)` | N/A |
`max-width: calc(768px - 0.02px)` | `max-width: calc(48rem - 0.02px)` | `--primer-viewportRange-narrow-viewport`
`max-width: calc(1012px - 0.02px)` | `max-width: calc(63.25rem - 0.02px)` | N/A

In a follow-up we might could try `--primer-viewportRange-narrowLandscape-viewport` for `1012px`? It matches the `63.25rem` but it also includes `max-height` + `orientation: landscape`. So we'd have to check if it causes any side effects.

/cc @langermank @vdepizzol